repair_order_product_by_lot/readme/USAGE.md

```
@@ -0,0 +1,11 @@
+- Create/edit a product and set traceability by 'By Lots' option.
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)